docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -757,14 +728,24 @@ sudo snap alias dotnet-sdk.dotnet dotnet
 
 Some distros require an additional step to enable access to the SSL certificate. See our [Linux Setup](https://github.com/dotnet/core/blob/master/Documentation/linux-setup.md) for details.
 
+## Docker and cgroup memory Limits
+
+As of Preview 3, running .NET Core 3.0 on Linux with Docker works better with cgroup memory limits. Running a Docker container with memory limits, such as with `docker run -m`, changes how .NET Core behaves.
+
+* Default GC heap size: maximum of 20mb or 75% of the memory limit on the container.
+* Explicit size can be set as an absolute number or percentage of cgroup limit.
+* Minimum reserved segment size per GC heap is 16mb, which reduces the number of heaps created on machines with a large number of cores and small memory limits.
+
+The GC change is the most critical part of our memory limits solution. It's also important to [update BCL APIs to honor cgroup settings](https://github.com/dotnet/corefx/issues/35638). Those changes are not included in Preview 3 but will come later. We’d appreciate feedback on which BCL APIs are most important to update first ([another example](https://github.com/dotnet/corefx/issues/32748)).
+
 ## GPIO Support for Raspberry Pi
 
 Two new packages have been released to NuGet that you can use for GPIO programming.
 
 * [System.Device.Gpio](https://www.nuget.org/packages/System.Device.Gpio/0.1.0-prerelease.19078.2)
 * [Iot.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings/0.1.0-prerelease.19078.2)
 
-The GPIO Packages includes APIs for GPIO, SPI, I2C and PWM devices. The IoT bindings package includes [device bindings](https://github.com/dotnet/iot/blob/master/src/devices/README.md) for various chips and sensors, the same ones available at [dotnet/iot - src/devices](https://github.com/dotnet/iot/tree/master/src/devices).
+The GPIO packages include APIs for GPIO, SPI, I2C, and PWM devices. The IoT bindings package includes [device bindings](https://github.com/dotnet/iot/blob/master/src/devices/README.md) for various chips and sensors, the same ones available at [dotnet/iot - src/devices](https://github.com/dotnet/iot/tree/master/src/devices).
 
 The updated serial port APIs that were announced as part of .NET Core 3.0 Preview 1 are not part of these packages but are available as part of the .NET Core platform.
 
```

Comment:
We don't typically cover platform support on the what's new article. We usually cover versions supported in the prereqs topics instead.

Reply:
I'll clean this up in the next update.